packages/webpack-plugin/src/webpack-config-stylable-excludes.ts

```diff
@@ -54,3 +54,31 @@ function safelyWalkJSON(
         }
     }
 }
+
+export function bundleLibs(config: Configuration, packages: string[], isServer: boolean) {
+    if (isServer) {
+        if (Array.isArray(config.externals) && config.externals.length === 1) {
+            const nextExternal = config.externals[0];
+
+            if (typeof nextExternal === 'function') {
+                config.externals = [
+                    async (ctx: any) => {
+                        for (const pack of packages) {
+                            if (ctx.request.startsWith(pack)) {
+                                return false;
+                            }
+                        }
+                        return nextExternal(ctx, (data) => {
+                            (data as any).resolve();
+                        });
+                    },
+                ];
+            } else {
+                throw new Error(
```

Comment:
If only called inside the internal else, then it will silently do nothing for the case that the config is an object or a an array with more the a single item.

Reply:
fixed